Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -451,6 +450,18 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     /// Shows typing Indicator
     /// - Parameter typingMembers: typing members gotten from `channelController`
     open func showTypingIndicator(typingMembers: [_ChatChannelMember<ExtraData.User>]) {
+        if typingIndicatorView.isHidden {
+            Animate {
+                self.collectionView.contentInset.bottom += self.typingIndicatorViewHeight
+                self.collectionView.scrollIndicatorInsets.bottom += self.typingIndicatorViewHeight
+            }
+
+            let lastIndexPath = IndexPath(item: 0, section: 0)
+            if collectionView.indexPathsForVisibleItems.contains(lastIndexPath) {
+                scrollToMostRecentMessage()
+            }
```

Comment:
Is this here on purpose? Feels more like a leftover 🤔

Reply:
It's not a leftover, the goal of these lines of code are to scroll the message list to the bottom if the indicator is over the last cell. Example:

https://user-images.githubusercontent.com/12814114/120781187-b9b1e180-c520-11eb-8936-c016cd5ac552.mp4